extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/service/QuarkusMutableIdentifierGeneratorFactory.java

```diff
@@ -75,7 +76,13 @@ public Class getIdentifierGeneratorClass(final String strategy) {
     }
 
     @Override
-    public void injectServices(final ServiceRegistryImplementor serviceRegistry) {
-        original.injectServices(serviceRegistry);
+    public IdentifierGenerator createIdentifierGenerator(GenerationType generationType,
+            String generatedValueGeneratorName, String generatorName, JavaType<?> javaType, Properties config,
+            GeneratorDefinitionResolver definitionResolver) {
+        // FIXME this is going to bypass the capturing of the generator class:
+        //  is it a problem? can we deal with it?
```

Comment:
Not sure I understand the problem, but adding a comment anyway so that this doesn't get lost.

Reply:
I'll remove it - it's an old comment I had for myself and later figured my concern wasn't valid. Shouldn't have been included in the final branch :)